### PR TITLE
new linux makefile for Android portages

### PR DESCRIPTION
**Pull request by jeangit**
It's for compiling/creating apk from linux.
The principles are the same than the windows Makefile, as it's based on this.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Ignore generated files
 # ...
 
+.gitignore
```
Comment (raysan5) [inline .gitignore]: I think it shouldn't be here...

**Comment by raysan5** [comment]
@jeangit thanks! great addition, I added a minor review and for the moment I prefer to keep Windows named as `Makefile.Android`.

File: .gitignore (hunk 1 above)
Comment (jeangit) [reply .gitignore]: Can you remove it, or do you need i do another PR with this file removed ?

Comment (raysan5) [reply .gitignore]: send another PR please, also revert the Makefile.Android rename...

Comment (jeangit) [reply .gitignore]: well i tried to do another PR, but git is not my cup of tea. Feel free to grab Makefile.Android.linux directly, if you want.

**Pull request closed without merging**